specifyweb/frontend/js_src/lib/components/DataModel/businessRuleDefs.ts

```diff
@@ -202,6 +201,42 @@ export const businessRuleDefs: MappedBusinessRuleDefs = {
     },
   },
 
+  CollectionObjectGroup: {
+    fieldChecks: {
+      cogType: (cog): void => {
+        // The first COJO CO will automatically have isPrimary set to True when the COG type is 'consolidated'
```

Comment:
what does COJO mean?

Reply:
ah, CollectionObjectGroupJoin?

Reply:
yeah, CollectionObjectGroupJoin. we've been going ham on the acronyms with geo